### DISCUSSION TITLE
Why is a Custom Fields Default Rendering a List?

### DISCUSSION THREAD

**Discussion by olivdee** [joomla/joomla-cms#38715] [2. Cold Features]
I was wondering, why a custom fields default rendering is a list, so in (I guess) over 90% of the use cases one would need to create an override. Just as a quick example, a website needs a **subheading** and this gets implemented as a **custom field** with **Options > Display Options > Automatic Display: After Title**. As this renders via **\components\com_fields\layouts\fields\render.php** and in **line 64** `$output[] = '<li class="field-entry ' . $class . '">' . $content . '</li>';` and **lines 71-73** `<ul class="fields-container"><?php echo implode("\n", $output); ?></ul>` it is defined as a list and one has to create an override for this in order to render/output the subheading for example as a plain `<div>`. Maybe I am wrong, or I misunderstood here the concept, but I would think that a raw/plain output would come handy for most users, rather than outputting this as a list. Maybe even easily extended in the components xml by adding a selection for the output type? [RAW, DIV, LIST, DESCRIPTION LIST, SPAN]

**Comment by brianteeman** [comment]
Good idea

**Reply by olivdee** [reply]
What is the way to go here? Is somebody at the dev-team going to file a request? Or should I prepare something myself? Never done this before, are there best practices on how to contribute to the project?

**Reply by dgrammatiko** [reply]
There’s a pr: https://github.com/joomla/joomla-cms/pull/36265
